tools/cleaners.py

```diff
@@ -7,3 +7,14 @@ def clean_html(raw_html):
     regex = re.compile(u'<.*?>')
     clean_text = re.sub(regex, '', raw_html)
     return clean_text
+
+
+def parse_keywords_files(file_path):
+    """Convert keyword files into lists, ignoring # commented lines."""
+    keywords_list = []
+    with open(file_path, 'r') as f:
+        for line in f:
```

Comment:
in case you want it in one line `keywords_list = [line for line in f.read().split('\n') if line[0] != '#']`